packages/dredd/package.json

```diff
@@ -38,7 +38,7 @@
     "chai": "4.2.0",
     "clone": "2.1.2",
     "cross-spawn": "7.0.0",
-    "dredd-transactions": "9.1.0",
+    "dredd-transactions": "^9.1.1",
```

Comment:
Is this intentional adding the `^`?

Reply:
No, that's been added automatically after running `lerna version`.

Reply:
Changed that to an exact dependency version. Since Lerna bumps dependency versions of of a dependent package upon `lerna version`, it should not matter to include a range of dependencies or a fixed version number.